tests/frontend_selenium/tests/test_node.py

```diff
@@ -0,0 +1,240 @@
+import math
+import datetime
+import random
+from utils.utils import generate_inavalid_gateway, generate_inavalid_ip, generate_ip, generate_leters, generate_string, get_node_seed
+from pages.polka import PolkaPage
+from utils.grid_proxy import GridProxy
+from pages.node import NodePage
+
+#  Time required for the run (12 cases) is approximately 3 minutes.
+
+def before_test_setup(browser):
+    node_page = NodePage(browser)
+    polka_page = PolkaPage(browser)
+    grid_proxy = GridProxy(browser)
+    user = generate_string()
+    password = generate_string()
+    polka_page.load_and_authenticate()
+    polka_page.import_account(get_node_seed(),user,password)
+    node_page.navigate(user)
+    return node_page, polka_page, grid_proxy, password
+
+
+def test_node_page(browser):
+    """
+      Test Case: TC1216 - Navigate to farm node
+      Steps:
+          - Navigate to the dashboard.
+          - Select an account (with node).
+          - Click on Farm from side menu.
+      Result: User should be navigated to Farm Nodes page and see all his node.
+    """
+    node_page, _, grid_proxy, _ = before_test_setup(browser)
+    assert 'Your Farm Nodes' in browser.page_source
+    nodes = grid_proxy.get_twin_node(str(node_page.twin_id))
+    for node in nodes:
+        assert str(node['nodeId']) in browser.page_source
+
+
+# def test_search_node(browser):
```

Comment:
For the commented tests, please add a comment above each one mentioning that the these tests will remain commented until the following issues are resolved:

- Search 
   https://github.com/threefoldtech/tfgrid_dashboard/issues/392

- Sort
   https://github.com/threefoldtech/tfgridclient_proxy/issues/204

Reply:
Description updated.